releases/2022-08-26-releasenotes.md

```diff
@@ -0,0 +1,120 @@
+    --compressed
+```
+
+With the release of these new authorization options comes the end of the legacy authorization options previously supported.    
+
+# Sorting Results
+The ability to sort results based on a field was a frequent request from our users over the last few quarters.  We heard you and we did it!
+
+Now, any [indexed field](https://www.vendia.net/docs/share/data-modeling#indexes) can be used to sort the results returned by a `list` operation.  For example, if the `name` field of the `Inventory` entity is indexed, the list of products can be returned in order by name: 
+
+## GraphQL
+```graphql
+query OrderedInventoryList {
+  list_InventoryItems(limit: 100, order: {name: DESC}) {
+    _InventoryItems {
+      name
+      price
+      quantity
+    }
+  }
+}
+
+```
+
+## Client SDK
+Even better, the [Vendia Share Client SDK](https://www.vendia.net/docs/share/vendia-client-sdk) makes sorting extremely simple.
+
+```graphql
+const orderedProducts = await entities.inventory.list({ limit: 100, order: { name: 'DESC' } })
+```
+
+# Schema Evolution
+Because indexes can now be used to sort results, we've also made it easier to manage indexes (i.e. add and remove) during scheme evolution.  
+
+## Share CLI
+You can now remove an index, which was previously deemed a breaking change, using the `--allow-breaking-change=INDEX_REMOVED` flag available through the `share uni evolve` [CLI command](https://www.vendia.net/docs/share/cli/commands/uni#uni-evolve).  This allows a user to add or remove an index simply and easily, likely enabling filtering and sorting on the fields of most interest to your users and applications.
+
+## Share UI
+Likewise, the Share UI has been enhanced to allow schema evolution from the Schema view of a Uni.
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/85032783/186989144-3b7a7b3a-6d70-4a80-a084-a69ba0f79d9f.png"/>
+</p>
+<p align="center">Schema Evolution in Share UI</p>
+
+The Share UI view also includes a very helpful "diff" feature that allows you visually inspect a schema change priory to confirming the schema evolution.
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/85032783/186988253-4f150aa6-17dd-45e9-89eb-c0591d7dff07.png"/>
+</p>
+<p align="center">Schema Diffs in Share UI</p>
+
+# Control Plane and Data Plane Geo-Blocking
+To help protect our customers, Vendia Share's Control Plane and [all of its operations](https://www.vendia.net/docs/share/share-graphql-schema#vendia-share-graphql-schema-reference) are now *inaccessible to traffic* originating from a set of high-risk countries.  This technique, known as [geo-blocking](https://en.wikipedia.org/wiki/Geo-blocking).
+
+This same mechanism is now also in place on our [Enterprise customers'](https://www.vendia.net/pricing) Data Planes (i.e. protecting the GraphQL interfaces of your Nodes).  
```

Comment:
Is it worth noting that this feature is only available to Enterprise tier users?